Userland/Libraries/LibAudio/QOATypes.h

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Array.h>
+#include <AK/Forward.h>
+#include <AK/Math.h>
+#include <AK/Types.h>
+#include <math.h>
+
+namespace Audio::QOA {
```

Comment:
Perhaps a link to the "spec" in this file would be nice?
The only in source link seems to be the one in `QOALoader.h`

Reply:
I'll do another update with spec links and terms once there actually is a finalized spec (same for FLAC btw).